src/apiCalls.js

```diff
@@ -27,4 +27,21 @@ const getAllDestinations = () => {
   .catch(err => console.log(err.message))
 }
 
-export { getAllTravelers, getAllTrips, getAllDestinations };
+const postANewTrip = (resoData) => {
```

Comment:
This looks good! Don't forget to add an alert or message for the user end if there's an error.